examples/entity/importexport/README.txt

```diff
@@ -0,0 +1,56 @@
+####### Using MySQL as Datasource #######
+yum install mysql mysql-server mysql-libs
```

Comment:
please ignore the 1st line just saw you have mentioned it in the end..just see if you can generically get the oozie installation dir, to make the script better.